lib/castle-rb/client.rb

```diff
@@ -11,17 +11,30 @@ def initialize(request, response)
     end
 
     def identify(args)
-      @api.request('identify', args)
+      @api.request('identify', args) unless do_not_track?
     end
 
     def authenticate(args)
       @api.request('authenticate', args)
```

Comment:
Is this intentionally left working even if `do_not_track` is enabled? Unless this needs to be working I'd suggest delegating the `do_not_track!` call directly to the `@api` instance, blocking all requests